src/prefect/tasks/secrets/base.py

```diff
@@ -3,16 +3,37 @@
 from prefect.client.secrets import Secret as _Secret
 from prefect.core.task import Task
 from prefect.engine.result_handlers import SecretResultHandler
+from prefect.utilities.tasks import defaults_from_attrs
 
 
-class Secret(Task):
```

Comment:
To ensure the non-breaking of this we could keep the `Secret` name w/ a deprecation warning and formally drop in 0.11.0

Reply:
FYI I do keep the `Secret` name at the bottom of the file; I need to think about how I can raise a deprecation warning if it's used though 🧐

Reply:
ah I know, will update

Reply:
@joshmeek updated